GOVERNANCE.md

```diff
@@ -0,0 +1,183 @@
+# Pulumiverse Governance
+
+## ToC
+
+* [Roles and Responsibilities](#roles-and-responsibilities)
+  * [Users](#users)
+  * [Contributors](#contributors)
+  * [Collaborators](#collaborators)
+  * [Project Management Committee (PMC)](#project-management-committee-pmc)
+    * [PMC Onboarding](#pmc-onboarding)
+  * [Specialized Roles](#specialized-roles)
+* [Lazy Consensus](#lazy-consensus)
+* [PMC Removal](#pmc-removal)
+* [Transparency](#transparency)
+* [Election](#election)
+
+---
+
+## Roles and Responsibilities
+
+In order to have a smoothly running project, formal roles with corresponding
+responsibilities are established. A member of the community may have multiple
+roles.
+
+### Users
+
+*How to become one*: Use a Pulumiverse project
+
+Users are community members who have a need for the project. They are the most
+important members of the community: without them, the project would have no
+purpose. Anyone can be a user; there are no specific requirements. Users should
+be encouraged to participate in the life of the project and the community as
+much as possible. User contributions enable the project team to ensure that
+they are satisfying the needs of those users. Common user activities include
+(but are not limited to):
+
+* Advocating for use of the project
+* Informing developers of project strengths and weaknesses from a user’s perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Writing documentation and tutorials
+* Filing bug reports and feature requests
+* Participating on the discussion board
+
+Users who continue to engage with the project and its community will often find
+themselves becoming more and more involved. Such users may then go on to become
+collaborators and contributors, as described below.
```

Comment:
I don't understand this.
Are we trying to motivate people to get engaged with Pulumiverse with this?

I don't feel this is relevant for governance. Does that make sense?